weather_forecast.py

```diff
@@ -0,0 +1,65 @@
+from yr.libyr import Yr
```

Comment:
Denne filen vil kjøre ên gang og dø. Da får man været for de neste 8 dagene når systemet starter opp men aldri mer? Eller har du tenkt å kjøre den på tidsintervall i operativ systemet? Hvis det, så må det finnes utav hvordan det gjøres.